crates/infra/cli/src/commands/run/mod.rs

```diff
@@ -48,6 +48,11 @@ impl RunController {
 
         return Command::new("cargo")
             .arg("run")
+            .arg(if self.command == RunCommand::SolidityTestingSmoke {
```

Comment:
nit: not sure if an empty string arg is absorbed/displayed properly. I suggest conditionally adding it:

```rust
let mut command = Command::new("cargo").arg("run");

if self.command == RunCommand::SolidityTestingSmoke {
     command = command.flag("--release");
}

return command.arg(....rest....).run();
```

Reply:
Changed in https://github.com/NomicFoundation/slang/pull/592/commits/cab418983e897475517602e691e30dd6453b8318